src/components/10-atoms/checkbox/react/DemoCheckboxLabelAsChildren.jsx

```diff
@@ -10,7 +10,17 @@ const DemoCheckboxLabelAsChildren = () => {
 
   return (
     <AXACheckbox checked={checked} onChange={handleCheckboxChange}>
-      <p>or click here if you prefere</p>
+      <p>
+        click{' '}
```

Comment:
What is the {' '} for? Doesn't html insert a whitespace by itself here?

Reply:
No it doesn't for react. Prettier did this.